content/contributing/helping-out/how-you-can-help.md

```diff
@@ -0,0 +1,92 @@
+
+## Battle-testing Bevy
+
+Ultimately, Bevy is a tool that's designed to help people make cool games. Just by _using_ Bevy you're helping us catch bugs, prioritize new features, polish off rough edges, and promote the project. 
+
+If you're using Bevy, we want to hear from you! Don't hesitate to ask for support on [Github Discussions], [Discord], or [Reddit]. Once you've made something that you're proud of, feel free to drop a link, video, or screenshot in `#showcase` on [Discord]! If you release a game on [itch.io] we'd be thrilled if you tagged it with `bevy`.
+
+{% callout() %}
+Think you've found a bug, missing documentation, or a feature that would help you make better games? [File an issue](https://github.com/bevyengine/bevy/issues/new/choose) on the main `bevy` repo! Take a look at the chapter on [Reporting Issues](@/contributing/helping-out/reporting-issues.md) to learn more.
+{% end %}
+
+[GitHub Discussions]: https://github.com/bevyengine/bevy/discussions
+[Discord]: https://discord.gg/bevy
+[Reddit]: https://www.reddit.com/r/bevy
+[itch.io]: https://itch.io/games/tag-bevy
+
+## Reviewing Others' Work
+
+**One of the most valuable things you can do is review other people's work.** Bevy relies on community code review to keep the code quality high and the maintenance burden low. Absolutely anyone is welcome to leave reviews, and all reviews are treated with equal weight.
+
+You don't need to be an Elder Rustacean to be useful here: anyone can catch missing tests, unclear docs, logic errors, and so on. If you're new to Rust or Bevy, submitting reviews is great way to learn things and acquaint yourself with the internals of the engine.
+
+If you do have specific skills (e.g. advanced familiarity with `unsafe` code, rendering knowledge, or web development experience) or personal experience with a problem, that's fantastic! We suggest trying to prioritize those areas, to ensure we can get appropriate expertise where we need it.
+
+Not even our Project Leads and Maintainers are exempt from reviews! By giving feedback on this work (and related supporting work), you can help us make sure our releases are both high-quality and timely. 
+
+{% callout() %}
+Interested in reviewing but don't know where to start? Check out [Reviewing Pull Requests](TODO)!
+{% end %}
+
+## Joining a Working Group
+
+Bevy's active initiatives are organized into *temporary working groups*: public, open-membership teams where people work together to tackle a broad-but-scoped goal. Each working group coordinates through a dedicated forum-channel on [Discord], although they also create issues and may use project boards for tangible work that needs to be done.
+
+You should consider joining a working group if you're interested in contributing code but don't know where to start or what to work on. Choosing one and asking how to help can be a fantastic way to get up to speed and be immediately useful.
```

Comment:
```suggestion
You should consider joining a working group if you're interested in contributing but don't know where to start or what to work on. Choosing one and asking how to help can be a fantastic way to get up to speed and be immediately useful.
```
I don't think it has to be contributing code specifically, right? Of course, that's obviously welcome :P